web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/behandling/vedtak/dto/OppgaveDto.java

```diff
@@ -11,14 +11,14 @@
 import no.nav.foreldrepenger.behandlingslager.behandling.vedtak.OppgaveType;
 import no.nav.foreldrepenger.domene.typer.JournalpostId;
 
-public record OppgaveDto(OppgaveId oppgaveId, OppgaveType oppgavetype, List<Beskrivelse> beskrivelser, List<Dokument> dokumenter) {
+public record OppgaveDto(@NotNull OppgaveId oppgaveId, @NotNull OppgaveType oppgavetype, @NotNull List<Beskrivelse> beskrivelser, @NotNull List<Dokument> dokumenter) {
 
     public record OppgaveId(@NotNull @Digits(integer = 32, fraction = 0) @Min(1) @JsonValue String id) {
     }
 
-    public record Beskrivelse(String header, List<String> kommentarer) {
+    public record Beskrivelse(String header, @NotNull List<String> kommentarer) {
     }
 
-    public record Dokument(JournalpostId journalpostId, String dokumentId, String tittel) {
+    public record Dokument(@NotNull JournalpostId journalpostId, @NotNull String dokumentId, @NotNull String tittel) {
```

Comment:
Klarer ikke grave i bunns om tittel er Notnull her?

Reply:
Tittel er nullable - for dokument og journalpost. Arkivet krever ikke tittel, hverken på journalpost eller dokument.

Dessuten: Journalførings-UI bruker tjenester/dtos fra fpfordel, Avdelingsleder-UI og forsiden bruker Dtos fra fplos.
Til slutt: Felles beregning bruker Dtos fra ft-beregning (disse tenker vi flytte til kalkulus)